components/support/ktx/src/main/AndroidManifest.xml

```diff
@@ -2,4 +2,10 @@
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="mozilla.components.support.ktx" />
+    package="mozilla.components.support.ktx" >
+
+    <application>
+        <activity android:name=".MainActivity" />
```

Comment:
I think we should mark this as [exported false](https://developer.android.com/guide/topics/manifest/activity-element) as this is only for testing.

Reply:
Done